gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -282,6 +285,9 @@
               <p translate class="help-block">
```

Comment:
Just an idea, we could hide this regular help block when the "always enabled" message appears.  I think it's better to have less text overall and if the option cannot be changed, the explanation what it *would* do is not needed either.  If someone comes specifically to enable that option, they probably know why they want it and therefore have gotten their explanation somewhere else already.  I.e. the only important bit for them is that something prevents them from doing what they wanted.

Reply:
Tried doing that but I don't think it's really a good idea. This is because with no explanation present, the user will be forced to add a new folder of another type just to be able to know what the option does.